src/components/learner-credit-management/invite-modal/InviteSummaryCount.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from '@openedx/paragon';
+
+const InviteSummaryCount = ({ memberInviteMetadata }) => (
+  <Card className="d-flex px-3 py-2 rounded-0 shadow-none">
+    <Card.Footer className="p-0 justify-content-between" orientation="horizontal">
+      <span>
+        Total members to invite
```

Comment:
Based on what we talked about in office hours today, I think we should change this to "add" instead of invite